src/main/frontend/modules/outliner/tree.cljs

```diff
@@ -88,13 +88,16 @@
 
 (defn block-entity->map
   [e]
-  {:db/id (:db/id e)
```

Comment:
@tiensonqin This was changed so that [this test](https://github.com/logseq/logseq/pull/9613/files#diff-4a0f4a15e5ca8a3981de57507a2c55189c307ce911173ccb6e384b8d78d1414b) would pass as written. Some of the attributes were made conditional because they returning nil and failing the test e.g. `:block/left {:db/id nil}`